terraform/lib/dependabot/terraform/file_updater.rb

```diff
@@ -14,6 +14,7 @@ class FileUpdater < Dependabot::FileUpdaters::Base
       PRIVATE_MODULE_ERROR = /Could not download module.*code from\n.*\"(?<repo>\S+)\":/.freeze
       MODULE_NOT_INSTALLED_ERROR =  /Module not installed.*module\s*\"(?<mod>\S+)\"/m.freeze
       GIT_HTTPS_PREFIX = %r{^git::https://}.freeze
+      TF_PLATFORMS = "-platform=linux_amd64 -platform=darwin_amd64 -platform=windows_amd64"
```

Comment:
Could we make sure to include Apple Silicon support too?
```suggestion
      TF_PLATFORMS = "-platform=linux_amd64 -platform=darwin_amd64 -platform=darwin_arm64 -platform=windows_amd64"
```

Reply:
I am unsure if we can handle this atm, we would need to make sure that for all basic terraform providers we have the `darwin_arm64` build available. I can tell that we don't have the `linux_arm64` nor `darwin_arm64` available for all, that is why I excluded it for now from the static TF_PLATFORMS. To check some you can have a look at https://releases.hashicorp.com

Reply:
Mmm this probably needs to be configurable rather than hardcoded then I think - unless e.g. it can just add all available platforms

Reply:
The number of people using M1 MacBooks now is pretty huge - so I don't think this prevents the pain for "most" at this point really

I think I'll look into making a quick workflow which just updates the dependabot PRs to add on the other providers